src/xrpld/app/tx/detail/MPTokenIssuanceSet.cpp

```diff
@@ -180,9 +288,38 @@ MPTokenIssuanceSet::doApply()
     else if (txFlags & tfMPTUnlock)
         flagsOut &= ~lsfMPTLocked;
 
+    if (auto const mutableFlags = ctx_.tx[~sfMutableFlags])
+    {
+        for (auto const& f : mptMutabilityFlags)
+        {
+            if (*mutableFlags & f.setFlag)
+                flagsOut |= f.canMutateFlag;
+            else if (*mutableFlags & f.clearFlag)
+                flagsOut &= ~f.canMutateFlag;
+        }
+
+        if (*mutableFlags & tfMPTClearCanTransfer)
+        {
+            // If the lsfMPTCanTransfer flag is being cleared, then also clear
+            // the TransferFee field.
+            sle->makeFieldAbsent(sfTransferFee);
+        }
+    }
```

Comment:
nit: we can get rid of the asterisk
```suggestion
    if (auto const mutableFlags = ctx_.tx[~sfMutableFlags].value_or(0))
    {
        for (auto const& f : mptMutabilityFlags)
        {
            if (mutableFlags & f.setFlag)
                flagsOut |= f.canMutateFlag;
            else if (mutableFlags & f.clearFlag)
                flagsOut &= ~f.canMutateFlag;
        }

        if (mutableFlags & tfMPTClearCanTransfer)
        {
            // If the lsfMPTCanTransfer flag is being cleared, then also clear
            // the TransferFee field.
            sle->makeFieldAbsent(sfTransferFee);
        }
    }
```